introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-lst-format.ipynb

```diff
@@ -0,0 +1,733 @@
+    "\n",
+    "In addition, the endpoint configuration describes the instance type required for model deployment, and at launch will describe the autoscaling configuration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from time import gmtime, strftime\n",
+    "\n",
+    "timestamp = time.strftime('-%Y-%m-%d-%H-%M-%S', time.gmtime())\n",
+    "endpoint_config_name = job_name_prefix + '-epc-' + timestamp\n",
+    "endpoint_config_response = sage.create_endpoint_config(\n",
+    "    EndpointConfigName = endpoint_config_name,\n",
+    "    ProductionVariants=[{\n",
+    "        'InstanceType':'ml.p2.xlarge',\n",
+    "        'InitialInstanceCount':1,\n",
+    "        'ModelName':model_name,\n",
+    "        'VariantName':'AllTraffic'}])\n",
+    "\n",
+    "print('Endpoint configuration name: {}'.format(endpoint_config_name))\n",
+    "print('Endpoint configuration arn:  {}'.format(endpoint_config_response['EndpointConfigArn']))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create endpoint\n",
+    "Lastly, the customer creates the endpoint that serves up the model, through specifying the name and configuration defined above. The end result is an endpoint that can be validated and incorporated into production applications. This takes 9-11 minutes to complete."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "import time\n",
+    "\n",
+    "timestamp = time.strftime('-%Y-%m-%d-%H-%M-%S', time.gmtime())\n",
+    "endpoint_name = job_name_prefix + '-ep-' + timestamp\n",
+    "print('Endpoint name: {}'.format(endpoint_name))\n",
+    "\n",
+    "endpoint_params = {\n",
+    "    'EndpointName': endpoint_name,\n",
+    "    'EndpointConfigName': endpoint_config_name,\n",
+    "}\n",
+    "endpoint_response = sagemaker.create_endpoint(**endpoint_params)\n",
+    "print('EndpointArn = {}'.format(endpoint_response['EndpointArn']))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, now the endpoint can be created. It may take sometime to create the endpoint..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# get the status of the endpoint\n",
+    "response = sagemaker.describe_endpoint(EndpointName=endpoint_name)\n",
+    "status = response['EndpointStatus']\n",
+    "print('EndpointStatus = {}'.format(status))\n",
+    "\n",
+    "\n",
+    "# wait until the status has changed\n",
+    "sagemaker.get_waiter('Endpoint_Created').wait(EndpointName=endpoint_name)\n",
```

Comment:
Can you add the waiter logic from [here](https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/1P_kmeans_lowlevel/kmeans_mnist_lowlevel.ipynb) to this notebook?

Reply:
Added.